src/middleware/userAgentFilter.ts

```
@@ -0,0 +1,68 @@
+    .filter(Boolean)
+    .map((s) => new RegExp(s, "i"));
+  return [...DEFAULT_BLOCKED_PATTERNS, ...extras];
+}
+
+const BLOCKED_PATTERNS = buildBlockedPatterns();
+
+export function userAgentFilter(
+  req: Request,
+  _res: Response,
+  next: NextFunction,
+): void {
```

ESLint

[error] Replace `⏎··req:·Request,⏎··_res:·Response,⏎··next:·NextFunction,⏎` with `req:·Request,·_res:·Response,·next:·NextFunction`

(prettier/prettier)